src/components/EachContinent.js

```diff
@@ -0,0 +1,55 @@
+import React, { useEffect } from 'react';
+import { Link, useParams } from 'react-router-dom';
+import { useSelector, useDispatch } from 'react-redux';
+import { FaArrowLeft } from 'react-icons/fa';
+import NavBar from './shared/NavBar';
+import { fetchRegionDetails } from '../redux/countries';
+
+const EachContent = () => {
+  const regionDetails = useSelector((state) => state.entities.countries);
+
+  const { title } = useParams();
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(fetchRegionDetails(title));
+  }, []);
+
+  return (
+    <div>
+      <NavBar title={title} />
+      <div className="home-screen flex flex-column">
+        <ul className="flex space-btn home-data">
+          <li className="text-right">
+            <Link to="/" className="link flex justify-center align-center">
+              <FaArrowLeft />
+              Back to Home
+            </Link>
+          </li>
+          <li className="flex flex-column text-right">
+            <h2>{title}</h2>
+            <p>
+              {regionDetails.length}
+              Countries
+            </p>
+          </li>
+        </ul>
+      </div>
+      <div>
```

Comment:
- Hey, great adding the home and details page however you're yet to add the search functionality which can allow users to search by the continents name according to your app. As this is a core part of the project requirement, please, implement this feature by using the filter method on this line ` const regionDetails = useSelector((state) => state.entities.countries);` use `let` to implement the filter functionality. *This is just a suggestion. they are many ways to do this*.

:point_down: 
``` 
const [searchText, setYourFilterText] = useState('');
 let regionDetails = useSelector((state) => state.entities.countries);
  // const dispatch = useDispatch();

  regionDetails = regionDetails.filter((continent) => continent.name.toLowerCase().includes(searchText));

  const handleChange = (e) => {
    seYourFilterText(e.target.value.toLowerCase());
  };
```
:point_up: *That's just one way to do it.. you should tweak it to fit your needs*

- Also, you could add the search bar here below:

![search](https://user-images.githubusercontent.com/59397159/183240620-afcb389e-22cc-4038-b48d-abae2d85a9be.png)